models/intermediate/int_amplitude__date_spine.sql

```diff
@@ -17,8 +17,8 @@ with event_data as (
 -- create end_date_adjust variable
 {% if execute %}
 {% set end_date_query %}
-    -- select one day past current day
-    select  {{ dbt.dateadd("day", 1, dbt.date_trunc("day", dbt.current_timestamp())) }}
+    -- select one month past current day
+    select  {{ dbt.dateadd("month", 1, dbt.date_trunc("day", dbt.current_timestamp_backcompat())) }}
```

Comment:
I don't feel the need to make the calendar a month in the future. A day for the calendar will suffice.
```suggestion
    -- select one day past current day
    select  {{ dbt.dateadd("day", 1, dbt.date_trunc("day", dbt.current_timestamp_backcompat())) }}
```

Reply:
updated this